src/components/_table-list.scss

```
@@ -3,13 +3,15 @@
 // Browser support: IE9+, Safari6+, Firefox30+, Chrome30+
 //
 // Markup:
-//  <ul class="sb-table-list">
+//  <ul class="sb-table-list {{modifier_class}}">
 //    <li class="sb-table-list__item"> Element 1 </li>
 //    <li class="sb-table-list__item"> Element 2 </li>
-//    <li class="sb-table-list__item"> Element 3 </li>
+//    <li class="sb-table-list__item sb-table-list__item--selected"> Element 3 </li>
 //    <li class="sb-table-list__item"> Element 4 </li>
 //  </ul>
 //
+// .sb-table-list--without-line-divider - Remove line divider between elements
```

```suggestion
// .sb-table-list--borderless - Remove line divider between elements
```
instead of without-line-border, we can prefer this.